packages/client/src/types/routing.ts

```diff
@@ -69,7 +69,7 @@ export type ChainJSON = {
   logo_uri?: string;
   bech32_prefix: string;
   fee_assets: FeeAssetJSON[];
-  chain_type: string;
+  chain_type: ChainType | string;
```

Comment:
let's keep using enum
```suggestion
  chainType: ChainType;
```